tools/SetupFlow/DevHome.SetupFlow/Views/DevDriveView.xaml

```diff
@@ -55,159 +53,158 @@
                 HorizontalContentAlignment="Center"
                 VerticalContentAlignment="Center"
                 TabNavigation="Cycle">
-                    <StackPanel
-                        Margin="10 0 10 0">
-                        <Grid
+                <StackPanel
+                    Margin="10 0 10 0">
+                    <Grid
+                        Margin="0 0 0 12"

```

Comment:
Why the big change to this?

Reply:
Fixing messed up whitespace -- this section was too indented.

Reply:
It shouldn't be an actual code change, I wish the github diff were better at showing that.

Reply:
As long as it still works.